### PR TITLE
Book list

### PR DESCRIPTION
**Pull request by BJordi**
## Summary

- Created BooksList component.
- Created BooksListItem.
- Added service to get all the books from the API.
- Added integration with books API and rendered its data.

## Screenshot

image

### CODE REVIEW

File: src/services/books.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import api, { API_DEFAULT_HEADERS } from '@/config/api.js'
+
+export const getAllBooks = () => {
+  api.setHeaders(API_DEFAULT_HEADERS)
```
Comment (FrankIglesias) [inline src/services/books.js]: This header must be setted before

File: src/views/BooksListItem.vue [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<template lang="pug">
+  .book-detail-container
+    img.book-image(:src='book.image_url')
```
Comment (FrankIglesias) [inline src/views/BooksListItem.vue]: add alt

File: src/views/BooksListItem.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    max-height: 170px;
+    max-width: 200px;
+    width: 100%;
+  }
+
+  .title {
+    color: $black;
+    font-size: 20px;
+    font-weight: bold;
+    line-height: 1;
```
Comment (FrankIglesias) [inline src/views/BooksListItem.vue]: do you need this line-height: 1?

File: src/views/BooksListItem.vue [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    text-align: start;
+  }
+
+  .subtitle {
+    font-size: 16px;
+    line-height: 1;
+    text-align: start;
```
Comment (FrankIglesias) [inline src/views/BooksListItem.vue]: left?

File: src/views/BooksListItem.vue [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  }
+}
+
```
Comment (FrankIglesias) [inline src/views/BooksListItem.vue]: delete this line

File: src/views/Dashboard.vue [hunk 6]
```diff
@@ -1,14 +1,35 @@
 <template lang="pug">
   .dashboard-container
     nav-bar
+    books-list(:booksList='booksList')
 </template>
 
 <script>
 import NavBar from '@/components/NavBar'
+import BooksList from './BooksList'
+import { getAllBooks } from '../services/books.js'
```
Comment (FrankIglesias) [inline src/views/Dashboard.vue]: @/services

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: src/views/BooksListItem.vue (hunk 4 above)
Comment (BJordi) [reply src/views/BooksListItem.vue]: In this case is the same, but I will change it to left.

File: src/views/BooksListItem.vue (hunk 3 above)
Comment (BJordi) [reply src/views/BooksListItem.vue]: I will delete it.

File: src/services/books.js (hunk 1 above)
Comment (BJordi) [reply src/services/books.js]: Yeah, forgot to delete this! Thanks!

File: src/config/api.js [hunk 7]
```diff
@@ -1,10 +1,20 @@
 import { create } from 'apisauce'
+import LocalStorageService from '../services/localStorage'
```
Comment (FrankIglesias) [inline src/config/api.js]: @/services

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**